.env.example

```
@@ -0,0 +1,17 @@
+# The endpoint of the server's API that the client application will communicate with
+# E.g., http://localhost:5000/posts
+NEXT_PUBLIC_HOSTED_URL="http://localhost:3000"
+
+# The base URL for the Asgardeo root organization's API
+# E.g., https://api.asgardeo.io/t/your-org
+NEXT_PUBLIC_ASGARDEO_BASE_ORGANIZATION_URL="https://api.asgardeo.io/t/guardioinc"
+
+# The client ID for the Asgardeo Shared Application
+SHARED_APP_CLIENT_ID="<CLIENT ID OF CREATED APP>"
+
+# The client Secret for the Asgardeo Shared Application
+SHARED_APP_CLIENT_SECRET="<CLIENT SECRET OF CREATED APP>"
+
+# The name of the Asgardeo Shared Application
+SHARED_APPICATION_NAME="<Name OF THE CREATED APP>"
```

just for the sake of consistency :) 

```suggestion
SHARED_APPICATION_NAME="<NAME OF THE CREATED APP>"
```

---

Fixed